### ISSUE TITLE
Clase XML faltante

### ISSUE THREAD

**Issue by rrc220590** [dbadillasanchez/factura-electronica-API-Ministerio-de-Hacienda-CR#9]
En xadesSign.php se crea una instancia new XML(). 

Donde esta la clase XML? PHP tiene las clases SimpleXML y DOMDocument pero no encuentro ninguna llamada XML

**Comment by Lcarreno19** [comment]
Lograste firmar amigo ? solucionaste este punto ?

**Comment by rrc220590** [comment]
No, utilice un codigo en java para firmar el documento. Lo convoco mediante el cgi de php

**Comment by enzofjh** [comment]
quede igual, la clase XML no es al menos del core de PHP

**Comment by gmurillo0710** [comment]
Creo que lo que faltan son estas funciones --> generate_consecutive/ generate_key/ set_invoice/ get_token , eso del XML es un framework aparte el sowerphp.  también hay otro framework, el LibreDTE, aquí pareciera que deberían estar esas funciones, pero no las he visto. :(  Alguna idea?  Hacemos un grupo de ayuda mutua? :)

**Comment by agcgithub** [comment]
Para los compañeros que hayan avanzado.  Necesito una ayuda "for dummies".    En código PHP .

**Comment by Lcarreno19** [comment]
amigo @agcgithub .. yo logre el firmado en PHP tengo una libreria para firmar, y tengo la que realiza todo el proceso... si deseas puedes escribirme a ver en que acuerdo podemos llegar saludos luiscarreno1991@gmail.com

**Comment by gmurillo0710** [comment]
@agcgithub en crlibre obtienes el firmador e instrucciones de como comenzar en esto.
